src/constants.ts

```diff
@@ -250,7 +250,7 @@ After=osbuild-first-boot.service
 
 [Service]
 Type=oneshot
-ExecStart=/usr/local/sbin/custom-first-boot
+ExecStart=/bin/bash -x /usr/local/sbin/custom-first-boot
```

Comment:
This makes the shebang unnecessary. The newline is also not needed, it looks like bash does not care if a script does not have it and EOF will trigger the script as well.